features/202-session-management/api.md

```diff
@@ -63,8 +76,14 @@ For web SDK, the user agent would be controlled by browser, for example:
 
 The parsed user agent information is provided in best-effort basis, and its
 information may not be backward compatible.
-Some parsed user agent information may not be available, and empty string
-would be used instead.
+
+
+## SDK Extra Information Collection
+Extra information collection is disabled by default. If enabled, the collected
+information would be sent in cookie `SKYGEAR_SESS_INFO` as JSON encoded object.
+
+Extra information collection configuration would be persisted across app
```

Comment:
I think it should be a client side choice to supply what information to server? Why mention "configuration would be persisted across app restart"?

Reply:
Yes it is client side choice. Developer can, for example, update the device name once and no need to set it every time app starts.

Reply:
ic. Maybe say "Client is responsible for persisting the extra information configuration on its own" which is more clear? (App restart could meant the mobile app / web app or skygear app which is server side)

Reply:
I'll write 'Client SDKs should persist the extra information configuration locally', is that okay for you?

Reply:
Perfect.